frontend/views/containers/global-dashboard/NewsAndUpdates.vue

```diff
@@ -152,18 +33,80 @@ export default ({
   },
   data () {
     return {
-      dummyPosts
+      posts: [],
+      isLoading: true,
+      error: false,
+      errorMessage: ''
     }
   },
+  computed: {
+    ...mapGetters(['ourIdentityContractId', 'currentIdentityState'])
+  },
+  async mounted () {
+    await this.fetchNews()
+    await this.markNewsAsSeen()
+  },
   methods: {
-    humanDate
+    humanDate,
+    async fetchNews () {
+      try {
+        this.isLoading = true
+        this.error = false
```

Comment:
There is no need to manage a separate `this.error` variable. You can simply set `this.errorMessage = null` here (and have it as `null` initially in the `data ()` section.